index.html

```diff
@@ -119,8 +125,8 @@ <h3 class="Section-title">My Recent Works</h3>
          </div>
       </div>
    </section>
-   <section class="about-myself">
-      <div>
+   <section id="about-myself">
+      <div class="about-intro">
       <h3>About My Self</h3>
```

Comment:
- Please change the `<h3>About My Self</h3>` to `<h3>About  me</h3>` as it is more structurally correct and same as in the fogma design.
- [Optional] I noticed,  you have some missing background images, Please,  I will suggest you put them in their appropriate place. 

##### Figma design
![Screenshot from 2022-06-10 09-07-12](https://user-images.githubusercontent.com/58771507/173020899-6e7a75b9-78b4-49e1-8396-342c9fb8b664.png)

##### Your design 

![Screenshot from 2022-06-10 09-09-12](https://user-images.githubusercontent.com/58771507/173021224-ce7777de-8606-463a-9776-b26dff4c2610.png)